spec/grape/dsl/inside_route_spec.rb

```diff
@@ -118,9 +118,8 @@ def initialize
             .to raise_error(ArgumentError, 'Status code :foo_bar is invalid.')
         end
 
-        it 'raises error if unknow status code is passed' do
-          expect { subject.status 210 }
-            .to raise_error(ArgumentError, 'Status code 210 is invalid.')
+        it 'accepts unknown Fixnum status codes' do
+          expect { subject.status 210 }.to_not raise_error
```

Comment:
Can I be annoying? I think we also need an integration level test that says an endpoint actually returns this status. Thanks.